pages/api/members/index.js

```diff
@@ -7,10 +7,15 @@ const BYPASS_AUTH_ORIGINS = ['https://teamwaterloop.ca'];
 module.exports = async (req, res) => {
     await data.initIfNotStarted();
     if (req.method === 'POST') {
+        const authStatus = false;
```

Comment:
F my mistake I accidentally wrote this as `const` in my review make sure to make it `let` otherwise the reassignment will cause the code to break.